samples/client/petstore/typescript-axios/builds/composed-schemas/api.ts

```diff
@@ -217,11 +217,11 @@ export const DefaultApiAxiosParamCreator = function (configuration?: Configurati
         },
         /**
          * 
-         * @param {PetsFilteredPatchRequest | null} [petsFilteredPatchRequest] 
+         * @param {PetsFilteredPatchRequest} [petsFilteredPatchRequest] 
```

Comment:
FYI. This fix the incorrect `null` (as any type is incorrectly treated as null in openapi normalizer)

@TiFu (2017/07) @taxpon (2017/07) @sebastianhaas (2017/07) @kenisteward (2017/07) @Vrolijkx (2017/09) @macjohnny (2018/01) @topce (2018/10) @akehir (2019/07) @petejohansonxo (2019/11) @amakhrov (2020/02) @davidgamero (2022/03) @mkusaka (2022/04) @joscha (2024/10)